_layouts/default.html

```diff
@@ -4,12 +4,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">{% if '/index.html' == page.url or '/' == page.url %}
-    <meta name="description" content="JHipster is a fully Open Source, widely used application generator. Easily create high-quality Spring Boot + Angular/React projects!">
+    <meta name="description" content="JHipster is an open source, widely used application generator, made for modern full-stack developers.">
```

Comment:
"open source" or "Open Source"?

Reply:
Bruce Perens [agrees with you](https://perens.com/2017/09/26/on-usage-of-the-phrase-open-source/). According to [this reference](https://english.stackexchange.com/questions/157178/capitals-or-not-for-open-source), it's correct as I have it.

Reply:
If we write "web", then let's also write "open source"